sklearn/utils/_estimator_html_repr.py

```diff
@@ -403,6 +329,25 @@ def estimator_html_repr(estimator):
     html: str
         HTML representation of estimator.
     """
+    from sklearn.exceptions import NotFittedError
+    from sklearn.utils.validation import check_is_fitted
+
+    if not hasattr(estimator, "fit"):
+        status_label = "<span>Estimator is not fitted</span>"
+        is_fitted_css_class = ""
+    else:
+        try:
+            check_is_fitted(estimator)
+            status_label = "<span>Estimator is fitted</span>"
```

Comment:
```suggestion
        status_label = "<span>Not fitted</span>"
        is_fitted_css_class = ""
    else:
        try:
            check_is_fitted(estimator)
            status_label = "<span>Fitted</span>"
```

What do you think of reducing it to just fitted vs not fitted? Because this label appears on the thing representing the estimator we might not need to clarify that this refers to the estimator and instead display only the information we want to display (fit status).

Reply:
Looks good with me. Knowing that Estimator for the entire pipeline could maybe be misleading.